src/main/java/com/onarandombox/MultiversePortals/utils/PortalFiller.java

```diff
@@ -58,7 +60,17 @@ public boolean fillRegion(MultiverseRegion r, Location l, Material type) {
      */
     private void doFill(Block newLoc, int useX, int useZ, MultiverseRegion r, Material type) {
         if (isValidPortalRegion(newLoc.getLocation(), type)) {
-            newLoc.setType(type, false);
+            // we need to check if the fill material is nether_portal so we can rotate it if necessary
+            if (type == Material.NETHER_PORTAL) {
+                // for now we won't use physics because it causes errors
+                newLoc.setType(type, false);
+                if (useX == 0) {
+                    Orientable b = (Orientable) newLoc.getBlockData();
+                    b.setAxis(Axis.Z);
+                    // also don't use physics here
+                    newLoc.setBlockData(b, false);
+                }
+            } else newLoc.setType(type);
```

Comment:
please use
```java
else {
    newLoc.setType(type);
}
```
This is mostly just a general style nit pick but I would argue that this shortened style can lead to issues for future developers (forgetting to put in braces when adding an extra line and such). The only time I tend to think the short style is okay is
```java
if (something) return;
```

Reply:
Sure, I'll amend this today and keep it in mind for future PRs.